packages/ui/src/molecules/Alert/Alert.tsx

```diff
@@ -0,0 +1,73 @@
+import type { HTMLAttributes, MouseEvent, ReactNode } from 'react'
+import React, { forwardRef } from 'react'
+
+import Icon from '../../atoms/Icon'
+import Button from '../../atoms/Button'
+
+export interface AlertProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'onClose' | 'role'> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+
+  /**
+   * A React component that will be rendered as an icon.
+   */
+  icon?: ReactNode
+
+  /**
+   * Boolean that controls the close button.
+   */
+  dismissible?: boolean
+
+  /**
+   * Function that is triggered when the alert is closed.
+   */
+  onClose?: (event: MouseEvent) => void
+}
+
+const Alert = forwardRef<HTMLDivElement, AlertProps>(function Alert(
+  {
+    testId = 'store-alert',
+    children,
```

Comment:
Maybe you should do two versions for this component, pure/nonPure, such as the ones used in ModalContent/ModalContentPure.

This way you could export a more 'complete' version, but if any store needs a more customizable version, they can use it as well. What do you think?

Reply:
You are right, @saranicoly. 
I'm gonna follow this pure/nonPure approach. Thank you! 😉

Reply:
This makes me a little confused, because if they want to use the pure one, how could they? Since we're only exporting the complete one

Reply:
This makes me a little confused as well hahaha. But as we talked before, we could export the more complete version initially and if a store needs more customizations we can export the Pure afterward. That approach can also improve a little the final bundle size.

Reply:
It's exactly what @eduardoformiga said in his last comment